config/initializers/00_zeitwerk.rb

```diff
@@ -3,7 +3,7 @@
 # This file is a relic of how Lobsters used to manage API keys and some other config variables.
 # It only exists to give developers and sister sites an error message as they setup.
 
-if Rails.application.credentials.secret_key_base.blank?
+if ENV["SECRET_KEY_BASE_DUMMY"].blank? && Rails.application.credentials.secret_key_base.blank?
```

Comment:
`SECRET_KEY_BASE_DUMMY` is used in Rails's official Dockerfile template.

https://github.com/rails/rails/blob/97dac99b5e80cc0aab3764e86fbcd12f4c316112/railties/lib/rails/generators/rails/app/templates/Dockerfile.tt#L97